### ISSUE TITLE
more like a question... how to change the command that some buttons use?

### ISSUE THREAD

**Issue by eithanunal** [gh0stzk/dotfiles#325]
i've been just looking around the files and figuring out things  i love everything about the dotfiles and i like giving them my own touch. I wanted to change the update button adding the --noconfirm flag so that i don't have to press confirm and adding at the end of the command a exit so that once everithing has ended it closes by its own but i've had trouble with that.. it ends up just closing the kitty and opening a shell and doing strange things with the config files but that's the thing i wanna add.

**Comment by gh0stzk** [comment]
Hello, ok I assume that you are referring to when you left click on polybar in the available updates module, the window that opens to update, is that what you are referring to?

I explain to you how it works. When you left click on the available updates, the OpenApps file is executed with the command OpenApps --update that in turn executes the Term file which is responsible for distinguishing between kitty or alacritty in this case it is kitty and execute the following command : 

kitty --hold --class=FloaTerm -e Updates --update-system

That command is calling my Updates script and runs it with the Updates --update-system command

So basically you have to edit the Updates script and look for this function:

```
update_system() {
    paru -Syu --nocombinedupgrade
}

There change the paru command as you want.


```

**Comment by eithanunal** [comment]
thank you for answering. I added --noconfirm and exit in the next line with kill -9 $PPID
